Duality/Resources/Font.cs

```diff
@@ -876,14 +890,15 @@ public int PickTextGlyph(string text, float x, float y)
 			return -1;
 		}
 
-		private void ProcessTextAdv(string text, int index, out GlyphData glyphData, out Rect uvRect, out float glyphXAdv, out float glyphXOff)
+		private void ProcessTextAdv(string text, int index, out GlyphData glyphData, out Rect uvRect, out float glyphXAdv, out float glyphXOff, out float glyphYOff)
 		{
 			char glyph = text[index];
 			int charIndex = (int)glyph > this.charLookup.Length ? 0 : this.charLookup[(int)glyph];
 			this.texture.LookupAtlas(charIndex, out uvRect);
 
 			this.GetGlyphData(glyph, out glyphData);
 			glyphXOff = -glyphData.OffsetX;
+			glyphYOff = glyphData.OffsetY;

```

Comment:
This has nothing to do with kerning! All offsets can be both negative and positive, depending on the direction of the offset. They are not paddings, they are literally added to the position of a glyph when drawing it in a text block. 

What I was referring to in my initial comment is just that I think that their behavior should be similar in terms of which direction positive vs. negative values point to, and since the X offset is already defined with a minus sign in the above code, the Y offset should be too.